lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -415,6 +418,31 @@ export abstract class ClientApplication {
         });
     }
 
+    private trackPageVisibilityWithMeasurement():void {
```

Comment:
I wonder if this is something we can do in the performance client itself so we don't need to have logic dependent on the truthiness of the ssoSilent/acquireTokenByCode variables?

Reply:
I would rather keep this method here as it belongs to the browser lib only.